test/integration/pack/test_pack.py

```diff
@@ -578,6 +578,54 @@ def test_pack_tgz_git_version_compat(tt_cmd, tmpdir):
     assert os.path.isfile(package_file)
 
 
+@pytest.mark.slow
+def test_pack_tgz_git_version_compat_with_instances(tt_cmd, tmpdir):
+    shutil.copytree(os.path.join(os.path.dirname(__file__), "test_bundles"),
```

Comment:
Can we copy only the required bundle, not all bundles?

Reply:
Thanks for the comment, yes, we can copy only the required bundle. Fixed.

Reply:
This issue was present in some of the other `pack` tests too. Fixed it in the additional commit.